content/en/docs/setup/production-environment/dockershim.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Dockershim Deprecation FAQ
+content_type: concept
+aliases: [ '/dockershim']
+weight: 40
+---
+
+This document goes over some frequently asked questions and their answers. For
+more detail on the deprecation of Docker as a container runtime for Kubernetes
+kubelets, and what that means, check out the blog post
+[Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/)
+
+### Can I still use Docker in Kubernetes 1.20?
+
+Yes, the only thing changing in 1.20 is a single warning log printed at [kubelet]
+startup if using Docker as the runtime.
+
+[kubelet]: /docs/reference/command-line-tools-reference/kubelet/
+
+### When will dockershim be removed?
+
+Given the impact of this change, we are using an extended deprecation timeline.
+It will not be removed before Kubernetes 1.22, meaning the earliest release without
+dockershim would be 1.23 in late 2021, approximately one year from now. We will
+be working closely with vendors and other ecosystem groups to ensure a smooth
+transition and will evaluate things as the situation evolves. 
+
+### Are there examples of folks using other runtimes in production today that I can use to make the case to move forward with this change?
+
+The [kind] project has been using containerd for some time and has seen an
+improvement in stability for its use case. Kind and containerd are leveraged
+multiple times every day to validate any changes to the Kubernetes codebase. Other
```

Comment:
Maybe we want to explicitly say that it's part of release signal?

Reply:
I don't think most end users will really get "release signal"

Reply:
Something along the lines of "we check this before we released Kubernetes".

i.e., it's Community-validated for the artifacts we produce is a stronger message over it only being "leveraged".